src/CG/CG_elem.cpp

```diff
@@ -1876,9 +1876,7 @@ bool intersectSegmentPlane( array3D const &Q1, array3D const &Q2, array3D const
     array3D n = Q2 - Q1;
     n /= norm2(n);
 
-    array3D xP;
-    if ( intersectLinePlane(Q1, n, P2, n2, xP, distanceTolerance) && intersectPointSegment(xP, Q1, Q2, distanceTolerance) ) {
-        P = xP;
+    if ( intersectLinePlane(Q1, n, P2, n2, P, distanceTolerance) && intersectPointSegment(P, Q1, Q2, distanceTolerance) ) {
```

Comment:
From documentation: 
> P intersection point if intersect, else unaltered

With this modification it will be altered even if there is no actual intersection between plane and segment, but only between line and plane, @ksamouchos isn't it?

Reply:
Correct! I have to change the documentation as well. This change has only to do with personal preferences. So, if you don't agree, I can retrieve the initial behavior of the function.

Reply:
This change doesn't seem related to the bug you are fixing. Please move it to another commit.

Reply:
If the change is not really needed for some reason, for compatibility issues on possible applications using this function, personally, I'd keep the original behavior.

Reply:
Done!